kong/conf_loader.lua

```diff
@@ -346,7 +346,9 @@ local function load(path, custom_conf)
   local s = pl_stringio.open(kong_default_conf)
   local defaults, err = pl_config.read(s)
   s:close()
-  if not defaults then return nil, "could not load default conf: "..err end
+  if not defaults then
+    return nil, "could not load default conf: "..err
```

Comment:
Granted this is about 1 liners, but I think we are zealously trying to play by the book here. 

Adding spaces around the `..` operator is still part of the same scope related to "style improvements of the codebase conditions", and avoids 2 commits to change the same line unnecessarily.